relay-server/src/actors/upstream.rs

```diff
@@ -374,12 +374,12 @@ pub struct UpstreamRelay {
 ///  2. `ResponseError` in all other cases.
 fn handle_response(
     response: Response,
-    update_rate_limits: bool,
+    intercept_status_errors: bool,
```

Comment:
Please mention this in the doc comment. It currently reads:

> Depending on the status code and details provided in the payload, one of the following errors can be returned:

We should update this to:

> If `intercept_status_errors` is set to `true`, depending on the status code and details provided in the payload, one of the following errors is returned:

Reply:
ok